redisinsight/ui/src/components/charts/donut-chart/DonutChart.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import type { Meta, StoryObj } from '@storybook/react-vite'
+
+import DonutChart from './index'
+
+const donutChartMeta = {
+  component: DonutChart,
+  args: {
+    width: 400,
+    height: 200,
+    name: 'test',
+    data: [
+      { value: 1, name: 'A', color: [0, 0, 0] },
```

Comment:
Nit: Let's put a bit more variant styles, because now it looks... black.

<img width="297" height="240" alt="Image" src="https://github.com/user-attachments/assets/f35ba687-9832-45f4-8b02-961aa01a0a22" />